<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPreference</class>
 <widget class="QDialog" name="DialogPreference">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>870</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>870</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>首选项</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>850</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>850</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>850</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title">
    <string>内存数据动态压缩</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <item>
     <property name="text">
      <string>禁用数据压缩</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>自适应压缩</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>轻度压缩</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中度压缩</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>高度压缩</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>700</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>700</width>
      <height>90</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>当加载超大的数据文件时,程序通常会占用比较多的内存,在这里可以设置程序读取文件时执行动态内存压缩的等级,压缩等级越高,打开同样的文件本程序占用的内存可能越低(视文件可压缩程度),同时压缩等级越高,文件读取速度越慢,为了加快文件读取速度,默认禁用压缩,当需要打开超大文件时请启用,动态压缩并不会改变原始文件,本功能仅仅是保障可以读取超大数据文件的一种手段,开启或者关闭压缩功能会重载当前打开的文件</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>850</width>
     <height>221</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>850</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>850</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title">
    <string>其他</string>
   </property>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;在这里选择程序打开时默认的视图模式&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <item>
     <property name="text">
      <string>标准视图</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>精简视图</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>默认视图模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>拖拽打开文件多实例</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;拖放文件到程序界面时,在当前实例打开新的文件还是新建一个实例打开&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <item>
     <property name="text">
      <string>始终使用当前实例打开文件</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>使用新实例打开新不同文件</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>始终使用新实例打开新文件</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>超大文件分页每页行数</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;针对大文件的分页每页行数,选择10万行每页,则记录数低于10万时不分页&lt;/p&gt;&lt;p&gt;每页能承载的最大行数和你的电脑内存有关,内存较低时建议配置10万行每页&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <item>
     <property name="text">
      <string>10万行</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20万行</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50万行</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100万行</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200万行(占用内存较大)</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>开启必填检测</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>生僻字补充字体</string>
    </property>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>238</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;如果设置全局字体后生僻字显示异常,可以安装字体到操作系统后在这里选择一个补充字体&lt;/p&gt;&lt;p&gt;请注意1：如果你选择的字体没有生僻字字形，则生僻字依然可能显示为方块或者不显示该字符&lt;/p&gt;&lt;p&gt;请注意2：默认情况下只有全局字体不包含的字才会调用该字体显示，如需全局配置使用某字体，请设置全局字体&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>238</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;如非必要，无需配置该选项，此处选择的字体会作用于整个软件界面&lt;/p&gt;&lt;p&gt;在未配置的情况下，Windows默认使用“微软雅黑”字体，macOS默认使用“萍方”字体，Linux默认使用“文泉驿正黑”字体&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>全局字体</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>字体大小</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;在这里选择程序打开时默认的视图模式&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <item>
     <property name="text">
      <string>标准字号</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>较大字号</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>更大字号</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>370</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>应用设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>370</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>放弃修改</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
